File: Lab_1_Plecak/ConsoleApp1/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Linq;

namespace ConsoleApp1
{
    internal class Item
    {
        public int weight;
         public int value;
       public int index;
        public Item() { weight = -1;value = 0; }
        public Item(int x, int y,int z) { weight = x;value = y;index = z; }

        public override string ToString()
        {
            return "v: " + value + "  w: " + weight; 
        }
    }
}

File: Problem_plecakowy/src/main/java/org/example/item.java
package org.example;

public class item
{
   public int wartosc;
    public int waga;
    public int index;
    public item(int wartosc, int waga,int index)
    {
        this.wartosc = wartosc;
        this.waga = waga;
        this.index = index;
    }
    @Override
    public String toString()
    {
        StringBuilder sb=new StringBuilder();
        sb.append(index+". wartosc: "+wartosc);
        sb.append("----waga: "+waga+"\n");
        return sb.toString();
    }

}
